src/roswire/ros1/ros1.py

```diff
@@ -394,3 +405,28 @@ def playback(
             files=self.__files,
             delete_file_after_use=delete_file_after_use,
         )
+
+    def get_node_sources_for_source_package(
```

Comment:
name of this method is a bit lengthy. how about `node_sources_for_package` or `package_node_sources`?

Reply:
Done.